### DISCUSSION TITLE
[remix] Remix V2 `serverModuleFormat` as ESM when deploying to Vercel

### DISCUSSION THREAD

**Discussion by natac13** [vercel/vercel#10519] [General] [answered]
### Description

I am curious if remix users are able to update to `esm` for the latest remix v2 update using the `@vercel/remix` package when deploying to Vercel currently?

I checked the package and Im thinking that [this line](https://github.com/vercel/vercel/blob/eb40c4c4a0320e56f4167ee787929cfe55349252/packages/remix/src/build.ts#L311) is overriding any `serverModuleFormat` I set in `remix.config.json`

I was trying to figure out how to make these changes and just open a PR but Im not 100% sure all which needs to be changed. My apologizes.

**Comment by leerob** [comment, marked as answer]
Yes, we’re working on adding support for Remix v2. We’ll share updates as
they’re ready!

*--*
*Lee Robinson ***@***.*** <https://twitter.com/leeerob>)
▲ VP of Developer Experience at Vercel <https://vercel.com/>
